EXTENSIONS_CSS_MAP

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
@infonline could you revert the change here? Thanks!

Reply:
done